app/src/seed_script.py

```diff
@@ -0,0 +1,195 @@
+import os
+from .models import db, Course, Module, Topic, Lesson, Subject, User, Points, UserProgress, GradeEnum
+
+def create_file_paths(course_id, module_id, topic_id):
```

Comment:
Add an extra boolean parameter to the function, so that by setting it to False for text lessons, running create_file_paths() won't have to copy over an unneeded video and thumbnail:

```python
def create_file_paths(course_id, module_id, topic_id, is_video_lesson=True):
```